src/krylov_phiv.jl

```diff
@@ -247,11 +243,11 @@ function phiv!(w::AbstractMatrix{T}, t::Number, Ks::KrylovSubspace{T, U}, k::Int
         throw(ArgumentError("Cache must be a PhivCache"))
     end
     e, Hcopy, C1, C2 = get_caches(cache, m, k)
-    lmul!(t, copyto!(Hcopy, @view(H[1:m, :])))
+    lmul!(t, copyto!(Hcopy, H[1:m, :])) # For the moment copyto! doesn't work with views. lmul!(t, copyto!(Hcopy, @view(H[1:m, :])))
```

Comment:
what about `copy!`?

Reply:
Also `copy!` seems not to work.

Reply:
Yes I will do that.

Reply:
Since this allocates, can you put it behind an if statement to only apply to gpuarrays so that the CPU performance is unaffected, add a comment that links to that?